### ISSUE TITLE
Auto complete in new column editor

### ISSUE THREAD

**Issue by jisnajc** [finos/perspective#2052]
I have installed the below dependencies for perspective in the angular app.
But new column editor is working as expected. Its not showing the features like autocomplete etc.. Did I missed any dependencies?
<img width="547" alt="Capture" src="https://user-images.githubusercontent.com/68993854/207878753-7f83e366-48b9-4520-8d14-5949dc6969b7.PNG">
<img width="649" alt="Capture1" src="https://user-images.githubusercontent.com/68993854/207878915-cc09083d-85f1-46b3-a6f4-b59a9f36d478.PNG">
<img width="620" alt="Capture2" src="https://user-images.githubusercontent.com/68993854/207879082-e6736c7f-9c87-4a86-a0cc-ebf522792ea6.PNG">

**Comment by jisnajc** [comment]
Is there any updates on the query? Pls help to resolve the issue. It would be helpful if you are providing any working examples of perspective implementation in angular.

**Comment by sinistersnare** [comment]
Monaco is not used anymore. There is a custom editor implementation, and there are a few less features.

**Comment by jisnajc** [comment]
Ok.I have added the custom elements from webcomponents package in index.html as shown below.
<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/68993854/209128266-323fca27-0474-4668-896f-decaa34917cc.png">
And I am using versions  angular 12 and perspective 1.7.1 and web components 2.5.0 as below.
<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/68993854/209128630-bfbecc2f-f28b-4d76-90cc-1cbd7dc35f8a.png">
<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/68993854/209128784-ac26e350-88f3-4bff-a262-57572529b30c.png">
When I downloaded the working example from link https://github.com/bocharoviliyav/perspective-angular, the respective versions was 11, 0.6.2, 2.5.0. I tried upgrading that working solution to that of my application version. there also I am getting the same error. Posting the screenshot of error below.
